README.md

```diff
@@ -76,10 +76,16 @@ building on versions below the ones stated. To install on a device,
 you **must** have a valid Apple Developer account license and have
 added the relevant mobile provisioning profiles from the Developer
 portal to your Xcode organizer (see Apple's website on instructions on
-how to do this).
+how to do this).  The profile should have these **exact** details, and 
+should include a distribution certificate:
+
+```conf
+Name:	MIT App Inventor Development
+App ID: edu.mit.appinventor.*
```

Comment:
We do not want people using MIT in the identifiers for their testing. They should be using something a namespace they might reasonably control. Worst case, they could use a reverse DNS style entry based on their email address or their GitHub username.

Reply:
This does not work, in my (very limited) experience.  The file [`AICompanionApp.xcodeproj/project.pbxproj`](https://github.com/mit-cml/appinventor-sources/blob/df152767e9a083536656337297909cbc96a4d34d/appinventor/AICompanionApp.xcodeproj/project.pbxproj#L813) has `PRODUCT_BUNDLE_IDENTIFIER = edu.mit.appinventor.aicompanion3` and this is the App ID that must be included in the provisioning profile, otherwise Xcode reports:

![image](https://github.com/mit-cml/appinventor-sources/assets/5699667/16f24e7b-dfea-429a-b354-7a5048f88a24)

and the `ant ios` build fails with:

![image](https://github.com/mit-cml/appinventor-sources/assets/5699667/940d64c6-4e14-4433-b9b7-c68bd57262b6)

Alternatively, we could include instructions to update AICompanionApp.xcodeproj/project.pbxproj in multiple places to a namespace that they control. What do you think @ewpatton ?